proto/vmcasfs.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+
+package vmcasfs;
+
+import "proto/remote_execution.proto";
+
```

Comment:
nit: newlines after field definition, comment what messages are for if it's not obvious

// This message does foo.
message Foo {
  // comment 1
  string field = 1;

  // comment 2
  string field = 2;
}

Reply:
Done.